packages/jahia/ipsec.yml

```
@@ -86,6 +86,25 @@ actions:
           exit 1
         fi
 
+        # check if right subnets overlap with env's subnet
+        pip install ipconflict
+
+        infra_subnet=192.168.0.0/16
+        subnets_file=$(mktemp)
+
+        cat > $subnets_file << END
+        $infra_subnet
+        END
+        
+        grep 'rightsubnet' $conf_tmpfile | grep -v '#' | grep -o "[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\/[0-9]\{1,\}" | uniq >> $subnets_file
```

```suggestion
        grep 'rightsubnet' $conf_tmpfile | grep -v '#' | grep -Eo "([0-9]{1,3}\.){3}[0-9]{1,3}(\/[0-9]{1,})?" | uniq >> $subnets_file
```
imo, grouping the first 3 bytes and using grep's switch `-E` in order to not have to backslash brackets will help readability for this regex. As the network mask bits is optional (strongswan will assume it's `/32` then), I also update this part of the regex.

---

Fixed